yucca/pipeline/task_conversion/Task012_BraTS21.py

```diff
@@ -89,9 +89,12 @@ def convert(path: str, subdir: str = "brats21/training_data"):
         target_imagesTr,
         target_imagesTs,
         ("FLAIR", "T1", "T1CE", "T2"),
-        labels={0: "background", 1: "necrotic tumor core", 2: "peritumoral edematous/invaded tissue", 3: "GD-enhancing tumor"},
-        regions_in_order=[[1, 2, 3], [1, 3], [3]],
-        regions_labeled=[1, 2, 3],
+        labels={0: "BG", 1: "NCR", 2: "ED", 3: "ET"},
```

Comment:
I get how this is nice for the regions, but I'm not a huge fan of the string matching here for two reasons:
1. It's important to have "1: "necrotic tumor core", 2: "peritumoral edematous/invaded tissue", 3: "GD-enhancing tumor"" so we can actually see what our labels mean, and
2. It might be quite annoying and fragile to match super long strings, which I think there should be space for